client/orm/internal/models/models_test.go

```diff
@@ -34,9 +34,9 @@ func (i *Interface) TableEngine() string {
 }
 
 func TestDbBase_GetTables(t *testing.T) {
-	c := NewModelCacheHandler()
```

Comment:
do not change this. All tests should create its own ModelCache to decouple from the global cache.